src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig

```diff
@@ -183,6 +184,153 @@
+                                            </tr>
+                                            <tr>
+                                                <td>content</td>
+                                                <td>{% trans %}The entry's content{% endtrans %}</td>
+                                                <td>=</td>
+                                                <td>{% trans %}Equal to…{% endtrans %}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>language</td>
+                                                <td>{% trans %}The entry's language{% endtrans %}</td>
+                                                <td>!=</td>
+                                                <td>{% trans %}Not equal to…{% endtrans %}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>mimetype</td>
+                                                <td>{% trans %}The entry's mime-type{% endtrans %}</td>
+                                                <td>OR</td>
+                                                <td>{% trans %}One rule or another{% endtrans %}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>readingTime</td>
+                                                <td>{% trans %}The estimated entry's reading time, in minutes{% endtrans %}</td>
+                                                <td>AND</td>
+                                                <td>{% trans %}One rule and another{% endtrans %}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>domainName</td>
+                                                <td>{% trans %}The domain name of the entry{% endtrans %}</td>
+                                                <td>matches</td>
+                                                <td>
+                                                    {% trans %}
+                                                    Tests that a <i>subject</i> is matches a <i>search</i> (case-insensitive).<br />
+                                                    Example: <code>title matches "football"</code>
+                                                    {% endtrans %}
+                                                </td>
+                                            </tr>
+                                        </tbody>
+                                    </table>
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+
                     {% if is_granted('ROLE_SUPER_ADMIN') %}
                     <div id="set5" class="col s12">
```

Comment:
It might be a side effect of your change (and some rebase) but could you set `set6` here instead of `set5`? Because we have 2 `set5` in the page and it breaks the layout because, if user is super admin, _Add a user_ will always be displayed.